src/components/select/_select.scss

```diff
@@ -279,8 +279,18 @@
     pointer-events: none;
   }
 
-  &[data-invalid] ~ .#{$prefix}--select__arrow {
-    bottom: 2.5rem;
+  &[data-invalid] ~ {
+    .#{$prefix}--select__arrow {
+      bottom: 2.4rem;
+    }
+
+    .#{$prefix}--select__invalid {
+      position: absolute;
+      right: rem(34px);
+      bottom: 2.15rem;
+      pointer-events: none;
```

Comment:
What does `pointer-events` do here?

Reply:
not sure why it was added initially but I had it in there probably from copying the original styles over. Since it is inherited anyway I removed it